projects/components/src/stepper/stepper-tab.component.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+import { IconType } from '@hypertrace/assets-library';
+
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../content/content-holder';
+
+@Component({
+  selector: `ht-stepper-tab`,
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabComponent extends ContentHolder {
+  @Input()
+  public label!: string;
+
+  @Input()
+  public icon?: string = IconType.Edit;
```

Comment:
if the goal of this icon is to have an icon with each step, I'd recommend not using a default or using a generic icon instead of `edit` which is for mutation actions.

Reply:
It was required to provide a default to override icon from angular material.

Reply:
Not sure what we can use for a generic icon.

Reply:
If we don't know what to use as a generic icon, that means we don't need one :)

Reply:
We need to override the default icon by material, or else it breaks. 
<img width="159" alt="image" src="https://user-images.githubusercontent.com/45289002/186148182-c2f700a9-69b2-42c0-ae44-818845ab29a6.png">